dts/bindings/sound/cirrus,cs42l73.yaml

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
```

Comment:
The filename should likely be cs47l63

Reply:
oops, my mistake when looking at Linux bindings